thalassa/api.py

```diff
@@ -59,6 +59,7 @@ def _resolve_ranges(x_range: tuple[float, float] | None, y_range: tuple[float, f
 def open_dataset(
     path: str | os.PathLike[str],
     normalize: bool = True,
+    source_crs: int = 4326,
```

Comment:
I guess we should force on the `int` type here, but I didn't know how to do that. 
I don't if we can use pyproj inside  (and also for the typing types)